CHANGELOG.md

```diff
@@ -56,6 +56,7 @@
 - Minor: Remove EmojiOne 2 and 3 due to license restrictions. (#1555)
 - Minor: Added `/streamlink` command. Usage: `/streamlink <channel>`. You can also use the command without arguments in any twitch channel to open it in streamlink. (#2443, #2495)
 - Minor: Humanized all numbers visible to end-users. (#2488)
+- Minor: Add tray icon (#2448, #1358)
```

Comment:
I'd make this a Major changelog entry and rephrase to something like `Added ability to minimize Chatterino to system tray` to make it more descriptive to an average end-user.